src/core/public/application/utils/get_app_info.ts

```diff
@@ -41,6 +41,7 @@ function getDeepLinkInfos(deepLinks?: AppDeepLink[]): PublicAppDeepLinkInfo[] {
   return deepLinks.map(
     ({ navLinkStatus = AppNavLinkStatus.default, ...rawDeepLink }): PublicAppDeepLinkInfo => {
       return {
+        ...rawDeepLink,
         id: rawDeepLink.id,
         title: rawDeepLink.title,
         path: rawDeepLink.path,
```

Comment:
NIT: now that we're spreading the initial deep link, we can remove the 

```ts
id: rawDeepLink.id,
title: rawDeepLink.title,
path: rawDeepLink.path,
```

part